src/NServiceBus.AcceptanceTests/Timeout/CyclingOutageTimeoutPersister.cs

```diff
@@ -95,7 +95,7 @@ public Task<TimeoutData> Peek(string timeoutId, ContextBag context)
             return Task.FromResult<TimeoutData>(null);
         }
 
-        public Task<TimeoutsChunk> GetNextChunk(DateTime startSlice)
+        public Task<TimeoutsChunk> GetNextChunk(DateTime startSlice, int maxChunkSize = Int32.MaxValue, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
I do agree that its ok to default in the EF case but we're not providing a generic api